### DISCUSSION TITLE
Elastic Search

### DISCUSSION THREAD

**Discussion by manhtd98** [saleor/saleor#11858] [Q&A] [answered]
### What I'm trying to achieve
…
Elastic Search is powerful tool for tools. In our case, i think it's nice solution for search.
Will you plan to use this in this project
...

### Other solutions I've tried and won't work
…

**Comment by peelar** [comment, marked as answer]
Hello @manhtd98 👋 

Just so you know, I converted your issue into a discussion.

---

Currently, we don't plan to use Elasticsearch directly in core API or Dashboard, but that doesn't close the door for the integration. Saleor offers an extendibility solution called [Apps](https://docs.saleor.io/docs/3.x/developer/extending/apps/key-concepts). By utilizing webhooks and API, you can build any logic you want on top of Saleor.

In the [saleor/apps](https://github.com/saleor/apps) repository, you can see all the currently maintained apps. In the [Discussions](https://github.com/saleor/apps/discussions), you can present your own app idea or an integration. In fact, we already [have plans](https://github.com/saleor/apps/discussions/2) for integrating Elasticsearch with one of our apps: [Search Hub](https://github.com/saleor/saleor-app-search) (it currently supports Algolia). 

If you'd like to help us ship it, you can describe your case for using Elasticsearch in [the discussion](https://github.com/saleor/apps/discussions/2), as well as propose features.
